bin/cron/calculate_workshop_survey_results

```diff
@@ -19,17 +19,20 @@ def main
 
   course_scores['CSP Local Summer Workshops'] = summarize_workshop_surveys(
     workshops: Pd::Workshop.in_state(Pd::Workshop::STATE_ENDED).where(course: Pd::Workshop::COURSE_CSP, subject: Pd::Workshop::SUBJECT_CSP_SUMMER_WORKSHOP),
-    facilitator_breakdown: false
+    facilitator_breakdown: false,
+    include_free_response: false
   )
 
   course_scores['CSD TeacherCon'] = summarize_workshop_surveys(
     workshops: Pd::Workshop.in_state(Pd::Workshop::STATE_ENDED).where(course: Pd::Workshop::COURSE_CSD, subject: Pd::Workshop::SUBJECT_CSD_TEACHER_CON),
-    facilitator_breakdown: false
+    facilitator_breakdown: false,
+    include_free_response: false
   )
 
   course_scores['CSP TeacherCon'] = summarize_workshop_surveys(
     workshops: Pd::Workshop.in_state(Pd::Workshop::STATE_ENDED).where(course: Pd::Workshop::COURSE_CSP, subject: Pd::Workshop::SUBJECT_CSP_TEACHER_CON),
-    facilitator_breakdown: false
+    facilitator_breakdown: false,
+    include_free_response: false
```

Comment:
Some of these lines are quite wide, beyond our 120 column limit. Can you break it up, either by querying workshops separately first, or something like
```ruby
workshops: (
  Pd::Workshop.in_state(Pd::Workshop::STATE_ENDED).where(
    course: Pd::Workshop::COURSE_CSP,
    subject: Pd::Workshop::SUBJECT_CSP_TEACHER_CON
  )
),
facilitator_breakdown: false,
...
```